extensions/ShowierData9978/html.js

```diff
@@ -0,0 +1,185 @@
+/* eslint-disable require-await */
+// BrowserJs
+/// <reference types="@turbowarp/types" />
+
+/**
+ * @typedef {import("@turbowarp/scratch-vm/")}
+
+ */
+
+((Scratch) => {
+  "use strict";
+
+  if (!Scratch.extensions.unsandboxed) {
+    throw new Error("HTML must run unsandboxed");
```

Comment:
This error will look rather ambiguous, so change it to "HTML Builder must run unsandboxed" to avoid confusion.